codecov.yml

```diff
@@ -0,0 +1,18 @@
+codecov:
+  require_ci_to_pass: true
+  branch: development
+
+# Pass only when the project has a 70% or greater coverage
+coverage:
+  status:
+    project:
+      default:
+        target: 70%
```

Comment:
can we also have patch set here to 70% pls

Reply:
Done but it's not failing due to no changes to rust code.